src/main/java/dev/openfeature/sdk/Value.java

```diff
@@ -19,11 +19,15 @@
  */
 @ToString
 @EqualsAndHashCode
-@SuppressWarnings({"PMD.BeanMembersShouldSerialize", "checkstyle:MissingJavadocType"})
+@SuppressWarnings({"PMD.BeanMembersShouldSerialize", "checkstyle:MissingJavadocType", "checkstyle:NoFinalizer"})
 public class Value implements Cloneable {
 
     private final Object innerObject;
 
+    protected final void finalize() {
+        // DO NOT REMOVE, spotbugs: CT_CONSTRUCTOR_THROW
+    }
```

Comment:
The other thing we could do here is make the class final. See https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ct-be-wary-of-letting-constructors-throw-exceptions-ct-constructor-throw

Reply:
I think making `Value` final makes sense. Let's do that if you also agree

Reply:
I think it's better to do that as well, and will probably satisfy sonar also.

My only concern was I think you could consider this a breaking change, though I highly doubt anyone would be extending Value, and it's not an expected use of the SDK.

Reply:
I would extend the class to help me handle my special `Object` type instead of being limited by `List`, `Instant` and so on.

Reply:
Ok so you think it's better to just add this final finalizer?

Reply:
Fine with me